tm/src/main/java/io/seata/tm/api/DefaultGlobalTransaction.java

```diff
@@ -88,6 +97,30 @@ public void begin(int timeout) throws TransactionException {
 
     @Override
     public void begin(int timeout, String name) throws TransactionException {
+        begin(timeout, name, Propagation.REQUIRED);
+    }
+
+    @Override
+    public void begin(int timeout, String name, Propagation propagation) throws TransactionException {
+        isUsed();
+        this.propagation = propagation;
```

Comment:
Do we need to restore this.propagation?

Reply:
> Do we need to restore this.propagation?

no,the same instance is allowed to be used only once